packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -0,0 +1,553 @@
+import { Icon, Modal, Tabs, ThemedScrollbars } from '@deriv/components';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+import { localize, Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { urlFor } from '_common/url';
+import routes from 'Constants/routes';
+import AccountCard from './account-card.jsx';
+
+import 'Sass/app/modules/account-types.scss';
+
+// TODO: this function is needed to be updated after end of all landing companies design
+const boxGenerator = ({
+    standpoint,
+    has_demo,
+    state,
+    setAccountTypeTabIndex,
+    redirectToMt5Real,
+    redirectToMt5Demo,
+}) => {
+    if (standpoint.malta && standpoint.maltainvest) {
+        // MLT/MF
+        return has_demo ? (
+            <Tabs
+                active_index={state.account_type_tab_index}
+                className='account-types__tabs'
+                fit_content
+                onTabItemClick={setAccountTypeTabIndex}
+                top
+            >
+                <div label={localize('Real accounts')}>
+                    <FinancialBox mt5OnClick={redirectToMt5Real} />
+                    <GamingBox mt5OnClick={redirectToMt5Real} />
+                </div>
+                <div label={localize('Demo accounts')}>
+                    <FinancialBox is_demo mt5OnClick={redirectToMt5Demo} />
+                    <GamingBox is_demo mt5OnClick={redirectToMt5Demo} />
+                </div>
+            </Tabs>
+        ) : (
+            <div>
+                <FinancialBox mt5OnClick={redirectToMt5Real} />
+                <GamingBox mt5OnClick={redirectToMt5Real} />
+            </div>
+        );
+    } else if (standpoint.iom && standpoint.maltainvest) {
+        // MX/MF
+        return has_demo ? (
+            <Tabs
+                active_index={state.account_type_tab_index}
+                className='account-types__tabs'
+                fit_content
+                onTabItemClick={setAccountTypeTabIndex}
+                top
+            >
+                <div label={localize('Real accounts')}>
+                    <FinancialBox mt5OnClick={redirectToMt5Real} />
+                    <GamingBox no_mt5 mt5OnClick={redirectToMt5Real} />
+                </div>
+                <div label={localize('Demo accounts')}>
+                    <FinancialBox is_demo mt5OnClick={redirectToMt5Demo} />
+                    <GamingBox is_demo no_mt5 mt5OnClick={redirectToMt5Demo} />
+                </div>
+            </Tabs>
+        ) : (
+            <div>
+                <FinancialBox mt5OnClick={redirectToMt5Real} />
+                <GamingBox no_mt5 mt5OnClick={redirectToMt5Real} />
+            </div>
+        );
```

Comment:
what is the difference between the first if and this else if?

Reply:
@negarn the first if is for MLT/MF users. this else if is for Only MLT users. based on [this flow](https://miro.com/app/board/o9J_kvOgFCg=/). they are not identical.

Reply:
@mahdiaryayi-fs I mean the code seems identical, am I missing something? I was hoping you would point out the difference in the code to me :D

Reply:
@negarn Yes their content is identical now, but I separated them so for example if we have a change in only MLT accounts, it doesn't affect the MLT/MF account.